### ISSUE TITLE
Error on viewing Layout tab in location full view.

### ISSUE THREAD

**Issue by sddb** [netgen-layouts/layouts-ezplatform#9]
The "Netgen Layouts" tab in location full view gives an error in EzP v3.0.5 / ng-layouts 1.1.4.1

Output from symfony serve:
```
[Web Server/PHP ] Jun 20 21:17:07 |ERROR| SERVER GET  (500) /admin/nglayouts/ezadmin/layouts/2 ip="::1"
[Application    ] Jun 20 21:17:07 |CRITI| REQUES Uncaught PHP Exception Twig\Error\LoaderError: "Template "@ezdesign/parts/table_header.html.twig" is not defined in "@admin/content/tab/nglayouts/rules.html.twig" at line 3." at /home/stefan/srv/ezplatform/vendor/twig/twig/src/Loader/ChainLoader.php line 99
```
If i change the include in vendor/netgen/layouts-ezplatform/bundle/Resources/views/themes/admin/content/tab/nglayouts/rules.html.twig and <same>/related_layouts.html.twig to
```
{% include '@ezdesign/ui/table_header.html.twig' with {
    'header_text': 'rules.title'|trans
} %}
```
IT does work.

**Comment by emodric** [comment]
Thanks for the report @sddb ! Will check it out as soon as possible!

**Comment by emodric** [comment]
@sddb Fixed in 1.1.4.2. Please note that the fix requires already released `netgen/layouts-core` v1.1.3
